Source/Letterbook.Adapter.Db/EntityConfigs/ConfigureProfile.cs

```diff
@@ -21,5 +21,8 @@ public void Configure(EntityTypeBuilder<Models.Profile> builder)
 			.WithOne(relation => relation.Follows)
 			.IsRequired();
 		builder.Property(profile => profile.Restrictions).HasColumnType("jsonb");
+		builder.Property(profile => profile.FollowersEstimate).HasDefaultValue(-1);
+		builder.Property(profile => profile.FollowingEstimate).HasDefaultValue(-1);
+		builder.Property(profile => profile.PostsEstimate).HasDefaultValue(-1);
```

Comment:
Setting default values in the entityConfig _should_ allow EFCore to tell the difference between previously set values and newly scaffolded ones, so the set values are preserved.